samples/dnn/ssd_mobilenet_object_detection.cpp

```diff
@@ -55,19 +57,32 @@ int main(int argc, char** argv)
     dnn::Net net = readNetFromCaffe(modelConfiguration, modelBinary);
     //! [Initialize network]
 
-    VideoCapture cap(parser.get<String>("video"));
+    if (net.empty())
+    {
+        cerr << "Can't load network by using the following files: " << endl;
+        cerr << "prototxt:   " << modelConfiguration << endl;
+        cerr << "caffemodel: " << modelBinary << endl;
+        cerr << "Models can be downloaded here:" << endl;
+        cerr << "https://github.com/chuanqi305/MobileNet-SSD" << endl;
+        exit(-1);
+    }
+
+    VideoCapture cap;
+    cap.open(parser.get<String>("video"));
     if(!cap.isOpened()) // check if we succeeded
     {
-        cap = VideoCapture(0);
+        int cameraDevice = parser.get<int>("camera_device");
+        cap = VideoCapture(cameraDevice);
         if(!cap.isOpened())
         {
-            cout << "Couldn't find camera" << endl;
+            cout << "Couldn't find camera: " << cameraDevice << endl;
```

Comment:
@catree, If user pass a wrong path to video file he has an unexpected error `Couldn't find camera: 0`. What do you think about the following logic: if parser has no `video` argument - open a camera stream. Otherwise, if video file didn't open - print a error like `cout << "Couldn't open image or video: " << parser.get<String>("video") << endl;`

Reply:
@dkurt I am ok with that.